### ISSUE TITLE
Distinguish between selected, not selected, and holiday for course dates

### ISSUE THREAD

**Issue by bwreid** [WikiEducationFoundation/WikiEduDashboard#2521]
![image](https://user-images.githubusercontent.com/1316902/53369476-28467480-3900-11e9-8ffd-14970eebe053.png)

Currently, when a screen reader is used to select dates on the course dates picker, it does not correctly distinguish between an unselected cell and a cell that is unselected _because_ it is a class holiday.

**Comment by hardy1334** [comment]
@bwreid  I guess you mean over here that in the above figure date 4 on Monday and 5 on Tuesday should look somewhat different that it currently is. Right?

**Comment by bwreid** [comment]
@hardy1334 It's not that they need to look different. When individuals use a [screen reader](https://en.wikipedia.org/wiki/Screen_reader) to navigate the site, there is not a distinction between the two dates you mentioned.

An ideal solution for this would be when a screen reader navigates to the 4th, it mentions something about the class being canceled. Does that help clarify?

**Comment by hardy1334** [comment]
@bwreid  I'm not quite clear with that. Suppose user comes to 4th part -

![Screenshot from 2019-03-14 23-34-24](https://user-images.githubusercontent.com/31733278/54380621-e9f7d600-46b1-11e9-850e-a079cca09aa3.png)

any suggestions what we can do over here

**Comment by ragesoss** [comment]
@hardy1334 this likely requires testing the site out with a screen reader to replicate the problem, and then researching accessibility techniques to figure out how to account for interactive elements with multiple possible states.

**Comment by hardy1334** [comment]
@ragesoss  @bwreid  Damn!!! It's so intresting I'll definetly start working over it. I guess currently i should implement for this **component** only not the entire website

**Comment by ragesoss** [comment]
Yeah, this a11y problem is specific to the calendar component with it's (I think) four different possible states for each date (disabled, selected, not selected, and holiday/canceled).
